src/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -135,13 +195,88 @@ INSTALLATIONS:
               TO: compressor1
             - FROM: compressor3
               TO: compressor1
-        OPERATIONAL_SETTINGS:
-          - RATES: [ 1000000, 6000000, 6000000 ]
-            INLET_PRESSURE: 50
-            OUTLET_PRESSURE: 250
-          - RATES: [ "$var.compressor1", 5000000, 5000000 ]
-            INLET_PRESSURE: 50
-            OUTLET_PRESSURE: 125
-          - RATES: [ 1000000, 5000000, 5000000 ]
-            INLET_PRESSURE: 50
-            OUTLET_PRESSURE: 125
+        STREAM_CONDITIONS_PRIORITIES:
+          pri1:
```

Comment:
I think we should provide an option to not specify priorities, just stream conditions directly.

We should also figure out how defaults could be set, default for a priority, default for all priorities, default for a compressor? Unit makes sense to set for all stream conditions in most cases I assume.